members/F000062.yaml

```diff
@@ -5,40 +5,48 @@ contact_form:
   steps:
     - visit: "https://www.feinstein.senate.gov/public/index.cfm/e-mail-me"
     - wait:
-        - value: 3
+        - value: 5
     - find:
-        - selector: "#field_0C5A1392-6512-4FA5-A213-EB7E1CBAF1AD"
+        - selector: input#field_0C5A1392-6512-4FA5-A213-EB7E1CBAF1AD
     - fill_in:
         - name: firstname
-          selector: "#field_0C5A1392-6512-4FA5-A213-EB7E1CBAF1AD"
+          selector: input#field_0C5A1392-6512-4FA5-A213-EB7E1CBAF1AD
           value: $NAME_FIRST
           required: true
         - name: lastname
-          selector: "#field_AC791FC0-5E57-4F5E-88D4-6EF9E730F470"
+          selector: input#field_AC791FC0-5E57-4F5E-88D4-6EF9E730F470
           value: $NAME_LAST
           required: true
         - name: street
-          selector: "#field_844E2E93-7496-4372-BB04-D0CF157684F0"
+          selector: input#field_844E2E93-7496-4372-BB04-D0CF157684F0
           value: $ADDRESS_STREET
           required: true
         - name: street2
-          selector: "#field_289B92DD-13E8-40E7-B141-1A02C6ACDCEA"
+          selector: input#field_289B92DD-13E8-40E7-B141-1A02C6ACDCEA
           value: $ADDRESS_STREET_2
           required: false
         - name: city
-          selector: "#field_C44B713C-04AD-411A-9744-6E7F2FF71898"
+          selector: input#field_C44B713C-04AD-411A-9744-6E7F2FF71898
           value: $ADDRESS_CITY
           required: true
+        - name: state
+          selector: select#field_1C62DB7F-A03B-4BC1-AE3F-816C2EC96B89
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          required: true
         - name: zip5
-          selector: "#field_64D2124B-A227-4F90-AD83-65DBC8175A69"
+          selector: input#field_64D2124B-A227-4F90-AD83-65DBC8175A69
           value: $ADDRESS_ZIP5
           required: true
         - name: email
-          selector: "#field_D065BDD9-7C1E-4D99-A50E-6A1ADAE8803B"
+          selector: input#field_D065BDD9-7C1E-4D99-A50E-6A1ADAE8803B
           value: $EMAIL
           required: true
+        - name: topic
```

Comment:
you don't need this either. The topic is getting selected here: https://github.com/NGPVAN/contact-congress/pull/358/files#diff-8f60da39bd935d35154e83adcc712abdR55

Reply:
got it. Will take this part out.